accelerometer/summariseEpoch.py

```diff
@@ -359,13 +360,10 @@ def calculateECDF(e, inputCol, summary, useRecommendedImputation):
     :return: Write dict <summary> keys '<inputCol>-ecdf-<level...>mg'
     :rtype: void
     """
-
-    ecdf1, step = np.linspace(1, 20, 20, retstep=True)  # 1mg bins from 1-20mg
-    ecdf2, step = np.linspace(25, 100, 16, retstep=True)  # 5mg bins from 25-100mg
-    ecdf3, step = np.linspace(125, 500, 16, retstep=True)  # 25mg bins from 125-500mg
-    ecdf4, step = np.linspace(600, 2000, 15, retstep=True)  # 100mg bins from 500-2000mg
-    ecdfXVals = np.concatenate([ecdf1, ecdf2, ecdf3, ecdf4])
-
+    filename = intensityDistributionVals[0]
```

Comment:
Hmm I don't follow this. `intensityDistributionVals[0]` is grabbing the first letter of the string -- in this case "a" from "accelerometer/intensityDistributionVals.txt". 
This is the error I got when running `python accProcess.py data/sample.cwa.gz --intensityDistribution True`:

![Captura de pantalla de 2021-01-28 08-03-00](https://user-images.githubusercontent.com/5551642/106131788-e2286a00-6141-11eb-8609-cc5f7ce18a70.png)

Reply:
This is weird because I thought this was horrible syntax, but I had to put it in to get it to work. I'll investigate this further...